src/app/InteractionModelEngine.cpp

```diff
@@ -664,7 +664,7 @@ bool InteractionModelEngine::TrimFabricForSubscriptions(FabricIndex aFabricIndex
          eventPathsSubscribedByCurrentFabric > perFabricPathCapacity ||
          subscriptionsEstablishedByCurrentFabric > perFabricSubscriptionCapacity))
     {
-        candidate->Abort();
+        candidate->Close();
```

Comment:
Not quite - I've actually removed the need to have separate `Abort` and `Close` and whittled it down into a single `Close` method that conceptually, just specifies a need to terminate any more work on the handler and take the requisite steps needed to clean it up.

That in turn, will automatically clean-up the exchange correctly. If there is a message that's still being re-transmitted that has no response, that will still remain in the re-trans table (based on the `ExchangeHolder::Release` logic). If a message is still being re-transmitted that does solicit a response, that will be forcibly removed since the handler presumably doesn't care about the response anymore.